.travis.yml

```diff
@@ -54,7 +54,10 @@ before_install:
     - export TRAVIS_COMMIT_MSG="$(git log --format=%B --no-merges -n 1)"
     # We don't use the travis saucelabs add on because it installs on every
     # test run, but we can manually use travis' saucelabs start/stop commands
-    - source <(curl -s https://raw.githubusercontent.com/travis-ci/travis-build/master/lib/travis/build/addons/sauce_connect/templates/sauce_connect.sh)
+    - if [[ -z "${SAUCE_USERNAME}" ]] && [[ -z "${SAUCE_ACCESS_KEY}" ]]; then export SAUCELABS=True; else export SAUCELABS=False; fi
+    - echo $SAUCELABS
+    - if [ -z $SAUCELABS = True ]; then source <(curl -s https://raw.githubusercontent.com/travis-ci/travis-build/master/lib/travis/build/addons/sauce_connect/templates/sauce_connect.sh); fi
```

Comment:
This conditional statement doesn't look like bash to me. Not enough quotes at least.